docs/content/policy-performance.md

```diff
@@ -333,6 +333,192 @@ Another way to get the same output as above would be the following:
 opa eval --data rbac.rego --profile-limit 5 --profile-sort num_eval --profile-sort num_redo --format=pretty 'data.rbac.allow'
 ```
 
+## Benchmarking Queries
+OPA provides CLI options to benchmark a single query via the `opa bench` command. This will evaluate similarly to
+`opa eval` but it will repeat the evaluation (in its most efficient form) a number of times and report metrics.
+
+
+#### Example: Benchmark rbac allow 
+Using the same [policy source as shown above](#example-policy):
+```bash
+$ opa bench --data rbac.rego 'data.rbac.allow'
+```
+
+Will result in an output similar to:
+```
++-------------------------------------------+------------+
+| samples                                   |      27295 |
+| ns/op                                     |      45032 |
+| B/op                                      |      20977 |
+| allocs/op                                 |        382 |
+| histogram_timer_rego_query_eval_ns_stddev |      25568 |
+| histogram_timer_rego_query_eval_ns_99.9%  |     335906 |
+| histogram_timer_rego_query_eval_ns_99.99% |     336493 |
+| histogram_timer_rego_query_eval_ns_mean   |      40355 |
+| histogram_timer_rego_query_eval_ns_median |      35846 |
+| histogram_timer_rego_query_eval_ns_99%    |     133936 |
+| histogram_timer_rego_query_eval_ns_90%    |      44780 |
+| histogram_timer_rego_query_eval_ns_95%    |      50815 |
+| histogram_timer_rego_query_eval_ns_min    |      31284 |
+| histogram_timer_rego_query_eval_ns_max    |     336493 |
+| histogram_timer_rego_query_eval_ns_75%    |      38254 |
+| histogram_timer_rego_query_eval_ns_count  |      27295 |
++-------------------------------------------+------------+
+```
+
+These results represent basically the best results than the query evaluation can be done on the local machine. All time spent
```

Comment:
😅 I'm afraid I still don't get it. (What's "best" here? 🤔What does the "than" mean?) -- I'll attempt to rephrase:

> These results capture metrics of `samples` runs, where only the query evaluation is measured. All time spent [...] is omitted.

Thanks for bearing with me!

Reply:
The "best" means its the best way to evaluate... unfortunately there are quite a few ways to get OPA to evaluate something from the perspective of a user, and they are not all equal performance-wise. The intent here is to convey that the evaluation that was done is the most optimal way to evaluate that query on the host.

`than` was just a typo..

I swapped in what you proposed, let me know if that works better.

Reply:
💡 I totally didn't get that "best" was referring to those over ways offered by OPA. (I'm fine either way; what we have now at least don't confuse _this reader_ anymore. 😃)